common/artifact_spawns/wc_artifact_spawns.txt

```diff
@@ -973,4 +972,141 @@ warblade_of_archimonde = {
 	artifacts = {
 		warblade_of_archimonde = { value = 1 }
 	}
+}
+orb_of_kiljaeden = {
+	max_amount = 1
+	spawn_date = 605.6.1
+
+	spawn_chance = {
+		value = 100
+		modifier = {
+			factor = 0
+			NOT = { year = 605 }
+		}
+	}
+
+	weight = {
+		value = 0
+		additive_modifier = {
+			value = 100
+			character = 340009						# Illidan
+		}
+	}
+	artifacts = {
+		orb_of_kiljaeden = { value = 1 }
+	}
+}
+scourge_bone_chimes  = {
+	max_amount = 3
+	spawn_date = 605.6.1
+	
+	spawn_chance = {
+		value = 100
+		modifier = {
+			factor = 0
+			 NOT = { year = 605 }
+		}
+	}
+	
+	weight = {
+		value = 0
+		additive_modifier = {
+			value = 99
+			character = 60003	# Arthas Menethil
+		}
+		additive_modifier = {
+			value = 1
+			religion_group = death_god
+			is_landed = yes
+		}
+	}
+	artifacts = {
+		scourge_bone_chimes = { value = 1 }
+	}
+}
+legion_doomhorn = {
+	max_amount = 5
+	spawn_date = 603.3.17
+
+	spawn_chance = {
+		value = 100
+		modifier = {
+			factor = 0
+			NOT = { year = 603 }
+		}
+	}
+
+	weight = {
+		value = 0
+		additive_modifier = {
+			value = 100
+			religion_group = fel_group
+			is_landed = yes
+		}
+	}
+	artifacts = {
+		legion_doomhorn = { value = 1 }
+	}
+}	
+serathil = {
+	max_amount = 1
+	spawn_date = 605.6.1
+	
+	spawn_chance = {
+		value = 100
+	}
+	
+	weight = {
+		value = 0
+		additive_modifier = {
+			value = 100
+			character = 52058						# Kor'Gall Stonemaul
+		}
+	}
+	artifacts = {
+		serathil = { value = 1 }
+	}
+}
+deathwhisper = {
+	max_amount = 1
+	spawn_date = 605.6.1
+	
+	spawn_chance = {
+		value = 100
+	}
```

Comment:
The same problem. Spawns before the spawn date.

Reply:
fixed